.ci/jenkins/Jenkinsfile.release

```diff
@@ -326,6 +326,9 @@ pipeline {
          }
         // send email for Sanity Checks
         stage('send email with build result'){
+            when{
+                 expression { reBuild == 'YES'}
+            }
```

Comment:
isn't this needed even if it runs for first time?

Reply:
es - but for default options this is always YES - it is only NO if you select it

Reply:
hmm, maybe to use different name of variable/property as it could be confusing for a reader of the definition?